gabrielgomesml/training_project_client
# Feature/Login-components

## What I did

- Create base of the project;
- Create Login components and page;
- Create Login hook;
- Create Private Routes;

## How to test

- First of all make sure you are running the server in your machine. You can check the steps here: https://github.com/gabrielgomesml/training_project_server/pull/1
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- After that run the command `yarn start` to start the project
- Try to access `localhost:8080/test` route. You should be redirect to login page;
- Log into an existing account and check the localstorage of the page. You should see the user info and a token;
- Last, click and the logout button and your credentials should be removed from localstorage.
